x/msgfees/types/msgs_test.go

```diff
@@ -77,3 +79,298 @@ func TestMsgAssessCustomMsgFeeValidateBasic(t *testing.T) {
 		})
 	}
 }
+
+func TestMsgAddMsgFeeProposalRequestValidateBasic(t *testing.T) {
+	authority := sdk.AccAddress("input111111111111111").String()
+
+	cases := []struct {
+		name     string
+		msg      *MsgAddMsgFeeProposalRequest
+		errorMsg string
+	}{
+		{
+			"Empty type error",
+			NewMsgAddMsgFeeProposalRequest("", sdk.NewCoin("hotdog", sdk.NewInt(10)), "", "", authority),
+			"msg type is empty",
+		},
```

Comment:
**Always** use named parameters when defining test cases. **Always**.
Without them, you can't make any changes to the test case struct without having to change every single test case, even when the test case isn't actually changing. They also make the tests easier to understand lower in the list where you don't have the struct definition on the screen anymore.

Reply:
Dang missed this.  I usually do the named parameters.  Don't know why I missed them here.